src/components/App/index.jsx

```diff
@@ -2,12 +2,14 @@ import { Route, Routes } from "react-router-dom";
 import MainPage from "../MainPage";
 import ErrorPage from "../ErrorPage";
 import ResultPage from "../ResultPage";
+import DetailPage from "../DetailPage";
 
 function App() {
   return (
     <Routes>
       <Route path="/" exact element={<MainPage />} />
       <Route path="/results" element={<ResultPage />} />
+      <Route path="/watch" element={<DetailPage />} />
```

Comment:
```suggestion
      <Route path="/watch" element={<VideoDetailPage />} />
```
조금 더 구체적인 네이밍이면 좋을 것 같아요!